server/web/src/components/dashboard/modules-panel.tsx

```
@@ -0,0 +1,402 @@
+  label,
+  icon: Icon,
+  status,
+  disabled,
+  onToggle,
+  tooltip,
+}: FeatureToggleProps) {
+  if (!status.supported) {
+    return (
+      <div
+        role="status"
+        aria-disabled="true"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<output>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)